content/running-amazon-ec2-workloads-at-scale/clone_github.md

```diff
@@ -18,4 +18,25 @@ In order to execute the steps in the workshop, you'll need to clone the workshop
 	cd ec2-spot-workshops/workshops/running-amazon-ec2-workloads-at-scale
 	```
 
-1. Feel free to browse around. You can also browse the directory structure in the **Environment** tab on the left, and even edit files directly there by double clicking on them.
\ No newline at end of file
+1. Feel free to browse around. You can also browse the directory structure in the **Environment** tab on the left, and even edit files directly there by double clicking on them.
+
+
+1. As you will neeed to modify the configuration files to refer to the identifiers of the resources created by the CloudFormation stack, run the following command to load the CloudFormation **Outputs** to environment variables. You will use these environment variables throughout the workshop to edit the files with *[sed](https://linux.die.net/man/1/sed)*.
+	```
```

Comment:
Hmmm Consider DRY ! 
A potential thing to reduce all this is to iterate over a set of values and extend the variables in a loop.

Note the code below is just an example, it does need to scape the right quotes and variable substitution but you get the idea. 

```bash
for output_value in "codeDeployBucket, instanceProfile, eventRule"
do 
export $(echo $output_value)=$(aws cloudformation describe-stacks --stack-name $stack_name --query 'Stacks[].Outputs[?OutputKey==`$output_value`].OutputValue' --output text)
done
```
```

Reply:
Agree; this is now fixed